### PR TITLE
aws-16-0-0

### PR DESCRIPTION
**Pull request by calvix**
aws release 16.0.0 with k8s 1.21 ant 2905.x.x flatcar

towards https://github.com/giantswarm/roadmap/issues/394

### CODE REVIEW

**Comment by calvix** [comment]
this wont work as release 16 has new flatcar where we experience issue with booting chart-operator  and its not yet solved

**Comment by calvix** [comment]
I am little bit puzzled why the cncf test here always fails, but it works if I execute it manually

**Comment by tfussell** [comment]
> I am little bit puzzled why the cncf test here always fails, but it works if I execute it manually

What's the failure? Some tests failed or something wrong with the tekton pipeline?

**Comment by calvix** [comment]
cncf PR is merged upstream, all tests were green including china, I rerun once more k8s-conformance locally and  all tests passed so ignoring the error from automation

the release is done from my side

@alex-dabija please go ahead and write a release notes and feel free to merge and announce

**Review by alex-dabija — APPROVED** [review]

File: aws/v16.0.0/announcement.md [hunk 1]
```diff
@@ -0,0 +1 @@
+**Workload cluster release v16.0.0 for AWS is available**. It provides support for Kubernetes 1.20. Further details can be found in the [release notes](https://docs.giantswarm.io/changes/workload-cluster-releases-aws/releases/aws-v16.0.0/).
```
Comment (calvix) [inline aws/v16.0.0/announcement.md:1]: its v 1.21

**Comment by calvix** [comment]
LGTM ( but can't approve my own PR) so feel free to merge

**Review by anvddriesch — APPROVED** [review]

**Pull request merged**